content/docs/engine-dev/scenepipeline/addingscenefiles.md

```diff
@@ -0,0 +1,43 @@
+---
+linkTitle: Adding Scene Files
+title: Adding Scene Files
+description: Describes the steps to take to add support for additional scene file formats.
+
+weight: 100
+---
+
+## Overview
+
+O3DE uses the [Open Asset Importer Library](./openassetimporter) to load scene files from a variety of scene formats, including files in the `fbx` format.
+
+This page covers the steps to take to enable additional scene file formats in the scene pipeline.
+
+## Enabling a scene format supported by [Open Asset Importer Library](./openassetimporter)
```

Comment:
I think it would be good to mention some scene types that are currently enabled and potentially point to https://github.com/o3de/o3de/blob/development/Registry/sceneassetimporter.setreg

Reply:
I link to https://www.o3de.org/docs/user-guide/assets/scene-settings/scene-format-support/ to avoid information duplication and minimize the maintenance cost of this documentation.

I didn't want to have the information spread out and duplicated in the documentation a bunch because it could easily lead to gaps of stale information. If someone enables Collada, I didn't want to introduce some hidden places that needed to be updated to mention that.